src/translate.py

```diff
@@ -4582,6 +4623,13 @@ def global_decls(self, fmt: Formatter, decls: Options.GlobalDeclsEnum) -> str:
                 if decls != Options.GlobalDeclsEnum.ALL and sym.initializer_in_typemap:
                     continue
 
+                if (
+                    sym.type.is_function()
+                    and name in self.local_functions
+                    and name not in forward_declares_needed
```

Comment:
Can you add `and decls != Options.GlobalDeclsEnum.ALL`, so that the filtering is disabled with `--globals=all`?